test/csuite/random_directio/util.c

```diff
@@ -100,6 +100,9 @@ copy_directory(const char *fromdir, const char *todir, bool directio)
 		testutil_check(__wt_snprintf(tofile, sizeof(tofile),
 		    "%s/%s", todir, dp->d_name));
 		rfd = open(fromfile, O_RDONLY | openflags, 0);
+		if (rfd < 0)
+			fprintf(stderr, "Open of %s failed with %d\n",
+			    fromfile, errno);
```

Comment:
Maybe `testutil_assertfmt(rfd >= 0, "Open of %s failed with %d", fromfile, errno);`

Reply:
Well that's a handy function. I didn't know it existed. Done. And since it will be permanent, I fixed the assert on the `wfd` open as well.